demos/java-hello-world/snap/snapcraft.yaml

```diff
@@ -7,7 +7,12 @@ grade: stable
 
 apps:
  hello:
-   command: usr/lib/jvm/default-java/bin/java oata.HelloWorld
+    command: usr/lib/jvm/default-java/bin/java oata.HelloWorld
```

Comment:
Most of our YAMLs use two spaces for indentation, this is now using three.

Reply:
interesting, on my editor I somehow counted 4!